README.md

```diff
@@ -58,6 +58,9 @@ break: true
 fastfail: false
 # Specify an indentation level in spaces to be used in this file
 indent: 4
+# Command to get user info/id
+# default as git user name
+idcommand: echo $TRAVIS_REPO_SLUG
```

Comment:
Should this maybe show the "default" command, just so that it's more clear what it does?